composer.json

```diff
@@ -22,20 +22,24 @@
         "chelout/laravel-relationship-events": "1.4.0",
         "doctrine/dbal": "2.13.2",
         "guzzlehttp/guzzle": "7.3.0",
+        "http-interop/http-factory-guzzle": "^1.0",
         "laravel/framework": "8.50.0",
         "laravel/horizon": "5.7.9",
-        "laravel/passport": "*",
         "laravel/nova": "*",
+        "laravel/passport": "*",
```

Comment:
This isn't related to your changes, but we should specify a more exact version here (the latest Passport is 10.1.3)

Reply:
Renovate will open a PR to do that after this is merged.

Reply:
I think it doesn't touch `*` packages.